jetbrains-core/src/software/aws/toolkits/jetbrains/utils/execution/steps/MessageEmitter.kt

```diff
@@ -64,7 +66,11 @@ class DefaultMessageEmitter private constructor(
     }
 
     override fun finishExceptionally(e: Throwable) {
-        emitMessage("$stepName finished exceptionally: ${ExceptionUtil.getNonEmptyMessage(e, ExceptionUtil.getThrowableText(e))}", true)
+        if (e is ProcessCanceledException) {
+            emitMessage(message("generate.step.canceled", stepName), true)
+        } else {
+            emitMessage(message("generate.step.failed", stepName, ExceptionUtil.getNonEmptyMessage(e, ExceptionUtil.getThrowableText(e))), true)
+        }
         if (hidden) return
         buildListener.onEvent(
             rootObject,
```

Comment:
should `FinishEventImpl` take a `SkippedResult` instead of a `FailureResult` on cancelled? (there doesn't seem to be a `CancelledResult`)

Reply:
Ya seems so looking at IJ sources, let me try it

Reply:
Cool, looks to be working